app/views/browse/imagestream.html

```diff
@@ -72,20 +72,21 @@
                     <td data-title="From">
                       <!-- this value can get long when its an ImageStreamImage. Use max width and class truncate to elide some
                            of the SHA1 for long values. They can still be copied even when elided. -->
-                      <div style="max-width: 400px;" class="truncate">
+
                         <span ng-if="!tag.spec.from"><em>pushed</em></span>
-                        <span ng-if="tag.spec.from">
-                          <span ng-if="!tag.spec.from._imageStreamName">
-                            {{tag.spec.from.name}}
-                          </span>
-                          <span ng-if="tag.spec.from._imageStreamName">
-                            <span ng-if="tag.spec.from._imageStreamName === imageStream.metadata.name">{{tag.spec.from._completeName}}</span>
-                            <span ng-if="tag.spec.from._imageStreamName !== imageStream.metadata.name" >
-                              <a ng-href="{{tag.spec.from._imageStreamName | navigateResourceURL : 'ImageStream' : (tag.spec.from.namespace || imageStream.metadata.namespace)}}"><span ng-if="tag.spec.from.namespace && tag.spec.from.namespace !== imageStream.metadata.namespace" >{{tag.spec.from.namespace}}/</span>{{tag.spec.from._imageStreamName}}</a>{{tag.spec.from._nameConnector}}{{tag.spec.from._idOrTag}}
+                        <!-- Wrap when < 768 & truncate when > 769 -->
```

Comment:
Really this comment should be in the less file not here. It will get out of sync if the style changes.